cluster/plan.go

```diff
@@ -81,7 +81,7 @@ func (c *Cluster) BuildKubeAPIProcess() v3.Process {
 		"--bind-address=0.0.0.0",
 		"--insecure-port=0",
 		"--secure-port=6443",
-		"--cloud-provider=",
```

Comment:
With PR #414, it would allow the following configuration in the cluster.yml file:
```
  extra_args:
    cloud-provider: aws
    configure-cloud-routes: false
```